bcipy/simulator/README.md

```diff
@@ -102,6 +102,28 @@ A simulation can be started using a graphical user interface.
 
 This provides a way to explore the file system when providing the parameters.json file, simulation model, and input data sources. After all required inputs have been provided the user can initiate the simulation from the GUI interface. The command line used to run the simulation are output to the console prior to the run making it easier to start subsequent simulations with the same set of arguments.
 
+## Replay Session
+
+The simulator also includes a Task for replaying a recorded session using a different signal model. This is used for testing if changes to a model result in more easily differentiated signals.
+
+This functionality currently has a different entry point.
+
+```
+(venv) $ python bcipy/simulator/task/replay_session.py -h
```

Comment:
What do you think about adding it to the console scripts? 

```
bcipy-replay 
```

Or it could use the one registered for sim with a different flag.

```
bcipy-sim --replay 
```

My concern is that for users who install BciPy (not clone) via pip, the path may be more challenging to access than a CLI interface.

Reply:
I agree, I do think we need a better way of running it. I'm still considering whether I can add it to the GUI, but I think that would be a separate PR. Maybe the quicker fix for this PR would be your second suggestion. I'll work on that.